pkg/cassandra/datacenter.go

```diff
@@ -389,6 +396,32 @@ func AddInitContainersToPodTemplateSpec(dcConfig *DatacenterConfig, initContaine
 	}
 }
 
+func AddVolumesToPodTemplateSpec(dcConfig *DatacenterConfig, extraVolumes api.K8ssandraVolumes) {
+	// Add and mount additional volumes that need to be managed by the statefulset
+	if extraVolumes.PVCs != nil {
```

Comment:
If `extraVolumes.PVCs` isn't nil then we can avoid the pointer dereferencing which I think makes the code less complex.

Reply:
done 👍